scripts/travis/android/linux-android.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/bash
+set -euxo pipefail
+
+[ "$ABI" = "armv7a" ] && host=$ABI-linux-androideabi || host=$ABI-linux-android
+sysroot="$ANDROID_NDK_ROOT/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/lib/$host"
+
+case "$ABI" in
+    "x86_64")
+        arch=$ABI
+        ;;
+    "i686")
+        arch=x86
+        ;;
+    "aarch64")
+        arch=arm64-v8a
+        ;;
+    "armv7a")
+        arch=armeabi-v7a
+        sysroot="$ANDROID_NDK_ROOT/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/lib/arm-linux-androideabi"
+        ;;
+    *) echo "unrecognized android ABI."; exit 1;;
+esac
+
+scons -Q clean
+
+scons -Q \
+    --enable-werror \
+    --disable-tools \
+    --enable-tests \
+    --build-3rdparty=libuv,openfec,speexdsp,cpputest \
+    --compiler=clang \
+    --host=$host$API
+
+# skip testing on ARM based platforms
+[ "$ABI" = "aarch64" ] || [ "$ABI" = "armv7a" ] && exit 0
+
+target_prefix="/data/local/tmp"
+
+device create --name roc-device-$host$API --image default --api $API --arch $arch
+device start --name roc-device-$host$API
+
+adb shell "su 0 mkdir -p ${target_prefix}/lib"
+adb push "$sysroot/libc++_shared.so" $target_prefix/lib/libc++_shared.so 1> /dev/null
+
+readarray -d '' tests < <(find bin/$host$API \
+        -name 'roc-test-*' -not -name 'roc-test-library' -print0)
```

Comment:
Why do we skip roc-test-library here? In qemu tests, we skip it because roc-test-library is multi-threaded, and qemu-arm has problems with threads.

Reply:
I was skipping roc-test-library according to linux-arm tests scripts.
On platform i686 with android API 24 roc-test-library never make any progress and timeout always expires; in other tested cases (i686 platform with android API 28-29 and x86_64 platform with all android APIs) roc-test-library was successful; so maybe we can skip roc-test-library only for i686 platform with API 24.

Reply:
> so maybe we can skip roc-test-library only for i686 platform with API 24

Yes, let's do it. Could you please create an issue for that? Would be great if you could also provide logs and backtrace. The backtrace can be captured by attaching to hanged test using gdb or by sending SIGABRT to specific tid (not pid).

Reply:
Sure I'll create an issue for that problem :+1: 
I've skipped roc-test-library for i686 platform for all APIs since it's not always successful.